packages/flutter/lib/src/material/scrollbar.dart

```diff
@@ -95,6 +96,13 @@ class Scrollbar extends StatelessWidget {
   /// {@macro flutter.widgets.Scrollbar.isAlwaysShown}
   final bool? isAlwaysShown;
 
+  /// Controls if the track will show in all descendant [Scrollbar] widgets.
```

Comment:
Why _descendant_ Scrollbar widgets? Does this affect more than just the scrollbar widget it is set on?

Reply:
Oh, this is a typo.